contracts/satoshi-bridge/src/api/token_receiver.rs

```diff
@@ -15,6 +15,7 @@ pub enum TokenReceiverMessage {
         output: Vec<TxOut>,
         max_gas_fee: Option<U128>,
         orchard_bundle_bytes: Option<String>,
+        expiry_height: Option<u32>,
```

Comment:
The `expiry_height` can be combined with `orchard_bundle_bytes` under one structure